front-end/src/app/tabs/dashboard/dashboard.page.html

```diff
@@ -75,7 +75,7 @@ <h1 *ngIf="!app.isInMobileMode()">{{ 'DASHBOARD.NEWS' | translate }}</h1>
           >
             <p class="ion-text-right">
               <app-statistics-button
-                *ngIf="app.user.isAdministrator && !editMode"
+                *ngIf="(app.user.isAdministrator || app.user.canManageDashboard) && !editMode"
```

Comment:
Lascerei solo agli admin, o dovremmo modificare in modo più pesante il Resource Controller delle statistiche